### PR TITLE
Standardisation-Prettier_Eslint_Husky_Lint-staged & Module aliases added

### PR DESCRIPTION
**Pull request by VISHNUDAS-tunerlabs**

### CODE REVIEW

File: src/route/routes.js [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-router.get('/jobs/jobList', jobList);
-router.post('/jobs/every', validate.everyAndOnce, every);
-router.put('/jobs/:jobname', validate.deleteAndUpdate, updateJob);
-router.post('/jobs/now', validate.nowAndCancel, now);
-router.post('/jobs/once', validate.everyAndOnce, once);
-router.post('/jobs/cancel', validate.nowAndCancel, cancel);
-router.delete('/jobs/:jobname', validate.deleteAndUpdate, deleteJob);
-router.post('/jobs/scheduleJob', validate.validation, scheduleJob);
-
+router.get('/jobs/jobList', jobList)
```
Comment (rakeshSgr) [inline src/route/routes.js:9]: @VISHNUDAS-tunerlabs please check this, refer other services

File: src/generics/constants/httpResponse.js [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
  * Description : all http response status
-*/
+ */
 
 module.exports = {
-    "OK" : "200",
-    "BAD_REQUEST" : "400"
-};
\ No newline at end of file
+	OK: 200,
```
Comment (rakeshSgr) [inline src/generics/constants/httpResponse.js:9]: @VISHNUDAS-tunerlabs constants folder should be in the inside src folder

File: src/models/log.js [hunk 3]
```diff
@@ -3,27 +3,32 @@
...
  * Date : 16-may-2022
  * Description : agenda exicution logs schema
-**/
+ **/
 // Dependencies
-const mongoose = require('mongoose');
-const connection = require("../configs/mongodb");
-const Schema = mongoose.Schema;
+const mongoose = require('mongoose')
+const connection = require('@configs/mongodb')
```
Comment (rakeshSgr) [inline src/models/log.js:9]: @VISHNUDAS-tunerlabs connection should available in global variable

Comment (rakeshSgr) [reply src/models/log.js:9]: @VISHNUDAS-tunerlabs refer mentoring service

File: src/services/agendaServices.js [hunk 4]
```diff
@@ -1,178 +1,152 @@
...
-    );
-  await agenda.start();
-  return jobDefCollection;
-});
-
+const jobsReady = agenda._ready.then(async () => {
+	const jobDefCollection = agenda._mdb.collection(configuration.definition)
+
+	jobDefCollection.toArray = () => {
+		const jobsCursor = jobDefCollection.find()
```
Comment (rakeshSgr) [inline src/services/agendaServices.js:23]: @VISHNUDAS-tunerlabs can we move all the queries to a single queries file for a model, refer https://github.com/ELEVATE-Project/mentoring/blob/master/src/db/entities/query.js

File: src/services/agendaServices.js [hunk 5]
```diff
@@ -1,178 +1,152 @@
...
+const addExicutionLog = async (job, status, resp, time) => {
+	let report = new log({
+		jobDetails: job,
+		runAt: time,
+		response: resp,
+		exicutionStatus: status,
+	})
+	await report
+		.save()
+		.then((doc) => {
```
Comment (rakeshSgr) [inline src/services/agendaServices.js:137]: @VISHNUDAS-tunerlabs remove then here

Comment (rakeshSgr) [reply src/services/agendaServices.js:137]: @VISHNUDAS-tunerlabs You may need to change other places also

File: src/services/jobs.js [hunk 6]
```diff
@@ -3,185 +3,186 @@
...
+		const jobs = await jobsReady
+		//checking job Definition is present
+		const jobExists = await utils.checkForDuplicateJobDefinition(job, jobs)
+
+		if (jobExists > 0) {
+			const scheduleJob = await scheduleOnce(job, agenda)
+			return scheduleJob
+		} else {
+			return {
+				message: responseMessage.JOB_DEFINITION_NOT_FOUND,
```
Comment (rakeshSgr) [inline src/services/jobs.js:93]: @VISHNUDAS-tunerlabs response should come from common function, check mentoring service

File: src/services/jobs.js [hunk 7]
```diff
@@ -3,185 +3,186 @@
...
+				success: false,
+				status: httpResponse.BAD_REQUEST,
+			}
+		}
+	} catch (err) {
+		return {
+			message: err,
+			success: false,
```
Comment (rakeshSgr) [inline src/services/jobs.js:177]: @VISHNUDAS-tunerlabs failure response should also come from common function, change in all the places

File: src/services/agendaServices.js [hunk 8]
```diff
@@ -1,178 +1,152 @@
...
+	agenda.define(name, async (job) => {
+		//needle is being implemented here
+
+		//adding header details
+		const options = {
+			headers: request.header ? request.header : {},
+		}
+
+		await needle(request.method, request.url, options)
+			.then(function (response) {
```
[the remark this answers is not in the document]
Comment (VISHNUDAS-tunerlabs) [reply src/services/agendaServices.js:43]: keeping then and removing await. Finding difficulty to determine needle error and success without then and error function.

File: src/services/agendaServices.js (hunk 4 above)
Comment (VISHNUDAS-tunerlabs) [reply src/services/agendaServices.js:21]: Agenda does not have any model for data operations..So can skip this?

**Pull request merged**